File: android/app/src/main/java/info/bagen/rust/plaoc/microService/user/ToyJMM.kt
package info.bagen.rust.plaoc.microService.user

import info.bagen.rust.plaoc.microService.sys.jmm.JmmMetadata
import info.bagen.rust.plaoc.microService.sys.jmm.JsMicroModule

class ToyJMM : JsMicroModule(
    JmmMetadata(
        id = "toy.bfs.dweb",
        server = JmmMetadata.MainServer(
            root = "file:///bundle", entry = "/toy.worker.js"
        ),
    )
) {}

File: next/dweb-browser/src/Platforms/iOS/WebModule/User/ToyJMM.cs
using DwebBrowser.WebModule.Jmm;
namespace DwebBrowser.WebModule.User;

public class ToyJMM : JsMicroModule
{
    public ToyJMM() : base(new JmmMetadata(
        "toy.bfs.dweb",
        new JmmMetadata.MainServer() { Root = "file:///bundle", Entry = "/toy.worker.js" }))
    {
    }
}

